### PR TITLE
Fixing timezone display for gethistory

### PR DESCRIPTION
**Pull request by g-theriot**

### CODE REVIEW

File: LEAF_Request_Portal/sources/System.php [hunk 1]
```diff
@@ -871,7 +871,7 @@ public function getPrimaryAdmin()
     public function setPrimaryAdmin()
     {
         $vars = array(':userID' => XSSHelpers::xscrub($_POST['userID']));
-        $resultArray = array('success' => false, 'response' => $res);
+        //$resultArray = array('success' => false, 'response' => $res);
```
Comment (abiola-adewuyi) [inline LEAF_Request_Portal/sources/System.php:874]: Can you remove this line if it's not being used?

File: LEAF_Request_Portal/admin/ajaxIndex.php [hunk 2]
```diff
@@ -281,6 +315,11 @@ function checkToken()
 
         $resHistory = $type->getHistory($itemID);
 
+        for($i = 0; $i<count($resHistory); $i++){
+            $dateInLocal = new DateTime($resHistory[$i]['timestamp'], new DateTimeZone('UTC'));
+            $resHistory[$i]["timestamp"] = $dateInLocal->setTimezone(new DateTimeZone($tz))->format('Y-m-d H:i:s T');;
```
Comment (abiola-adewuyi) [inline LEAF_Request_Portal/admin/ajaxIndex.php:320]: There's an extra ";" at the end of the line.

Comment (g-theriot) [reply LEAF_Request_Portal/admin/ajaxIndex.php:320]: Done

File: LEAF_Request_Portal/sources/System.php (hunk 1 above)
Comment (g-theriot) [reply LEAF_Request_Portal/sources/System.php:874]: Sure - done.

**Pull request merged**